appcues/src/main/java/com/appcues/data/mapper/step/StepMapper.kt

```diff
@@ -38,7 +38,6 @@ internal class StepMapper(
         return Step(
             id = from.id,
             content = responseContent.mapPrimitive(),
-            // TODO only one of each type, eg: @appcues/backdrop @appcues/skippable etc
```

Comment:
right, we have this handled now in TraitResponseAdapterFactory so I think all good 👍